### DISCUSSION TITLE
connectorsForWallets "Cannot use import statement outside a module"

### DISCUSSION THREAD

**Discussion by harshpandey002** [rainbow-me/rainbowkit#1274] [General]
```
import { connectorsForWallets } from '@rainbow-me/rainbowkit';
```

Just Importing connectorsForWallets throws this error, any idea why?

```
Server Error
SyntaxError: Cannot use import statement outside a module

This error happened while generating the page. Any console logs will be displayed in the terminal window.
...
```

I added `"type": "module"` to package.json but that just throws more errors and fails to start the server.

**Comment by DanielSinclair** [comment]
If you're still seeing this issue, please open an issue [here](https://github.com/rainbow-me/rainbowkit/issues) with a reproducible example.
